lc-hax/Scripts/Patches/EnemyPatch.cs

```diff
@@ -1,10 +1,33 @@
 #pragma warning disable IDE1006
 
+using GameNetcodeStuff;
 using HarmonyLib;
 
 namespace Hax;
 
-[HarmonyPatch(typeof(EnemyAI), nameof(EnemyAI.SyncPositionToClients))]
+[HarmonyPatch(typeof(EnemyAI))]
 class EnemyPatch {
+    [HarmonyPatch(nameof(EnemyAI.SyncPositionToClients))]
     static void Prefix(ref float ___updatePositionThreshold) => ___updatePositionThreshold = 0.0f;
+
+    // Fixes game bug where you can't aim anymore if an enemy dies while attempting to kill you.
+    [HarmonyPatch(nameof(EnemyAI.CancelSpecialAnimationWithPlayer))]
+    static void Prefix(ref PlayerControllerB ___inSpecialAnimationWithPlayer) {
+        if (Helper.LocalPlayer == ___inSpecialAnimationWithPlayer) {
+            Helper.LocalPlayer.disableLookInput = false;
+        }
```

Comment:
Very cool, but I think this can be set to `false` every time..? Let me know what you think.

```suggestion
Helper.LocalPlayer.disableLookInput = false;
```

Reply:
I think `CancelSpecialAnimationWithPlayer` gets called on every client when the AI dies, which is why I put this condition

Reply:
Yeah, but the patch only affects the local client and not anyone else.